### DISCUSSION TITLE
How to change LU-decomposition to GMRES?

### DISCUSSION THREAD

**Discussion by kont1901** [byuflowlab/FLOWUnsteady#203] [General]
Hello, I would like to test GMRES precomputed how to activate it in my Flowunsteady scripts?
Thanks

**Comment by EdoAlvarezR** [comment]
Hi Tenon!

@rymanderson and myself have been working on getting a matrix-free iterative solver implemented and integrated with the main branch of FLOWUnsteady (right now a prototype lives in a branch). Hopefully we will get something functional in about two weeks or so.

**Comment by kont1901** [comment]
Hi Edo,
Thanks for your reply. 
It's really appreciated.
